src/pools/abstract-pool.ts

```diff
@@ -103,12 +103,12 @@ export abstract class AbstractPool<
   /**
    * Constructs a new poolifier pool.
    *
-   * @param numWorkers Number of workers that this pool should manage.
+   * @param numberOfWorkers Number of workers that this pool should manage.
    * @param filePath Path to the worker-file.
    * @param opts Options for the pool. Default: `{ maxTasks: 1000 }`
    */
   public constructor (
-    public readonly numWorkers: number,
+    public readonly numberOfWorkers: number,
```

Comment:
This is not backward compatible, right ?

Reply:
Yes, I just thought I were safe because all tests passes :thinking:  
So I will add a note for that in the changelog

Reply:
Oh... I even didn't noticed that VSCode refactoring also changed it in the tests :sweat_smile:

Reply:
@pioardi I added backward compatibility in https://github.com/pioardi/poolifier/pull/139/commits/d66606eb9bdf359853779b181598cd7bddb0664a